table/two_level_iterator.cc

```diff
@@ -154,14 +154,8 @@ void TwoLevelIterator::Prev() {
 
 
 void TwoLevelIterator::SkipEmptyDataBlocksForward() {
```

Comment:
You added an explicit requirement for this function that `second_level_iter != nullptr`. It would be good to document this, preferably with `assert(second_level_iter.iter() != nullptr)`